StripeFinancialConnections/StripeFinancialConnections/Source/Native/NetworkingLinkSignupPane/NetworkingLinkSignupBodyFormView.swift

```diff
@@ -23,37 +29,50 @@ final class NetworkingLinkSignupBodyFormView: UIView {
     private lazy var verticalStackView: UIStackView = {
         let verticalStackView = UIStackView(
             arrangedSubviews: [
-                emailAddressTextField
+                formElement.view
```

Comment:
side-note this code gets further modified/polished in the next PR: https://github.com/stripe/stripe-ios/pull/2315